include/boost/beast/http/write.hpp

```diff
@@ -100,7 +100,7 @@ write_some(
 
     @param ec Set to indicate what error occurred, if any.
 
-    @return The number of bytes written to the stream.
+    @return The number of bytes read from the body.
```

Comment:
This is a breaking API change...

Reply:
Yes, and one that is overdue.

Reply:
The meaning of the return value used to be different from what it is now. We changed it because of user requests. If we change it again, then we are breaking those users.

Reply:
> The meaning of the return value used to be different from what it is now. We changed it because of user requests. If we change it again, then we are breaking those users.

Wasn't that for `async_read_some`? https://github.com/boostorg/beast/issues/1942

Reply:
Maybe. I think you should decide if this is a good change. And if not, what else we might do (if anything).